name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/WidgetFactory.java

```diff
@@ -58,6 +58,34 @@
                                     }) //
                                     .withBenchmarkDataSeries(false) //
                                     .build()),
+    
+    RATIO(Messages.LabelRatio, Messages.LabelStatementOfAssets, //
+                    (widget, data) -> MultiIndicatorWidget.<Double>create(widget, data) //
+                    .withDataSeries(delegate -> DataSeriesConfig.create(delegate)
```

Comment:
Jede Widget Konfiguration muss seinen Zustand in das XML serialisieren - und aus dieser XML Konfiguration muss das Widget seinen Zustand wiederherstellen. Programmatisch ist das eine ```Map<String, String>``` am ```Dashboard.Widget```. Die DataSeriesConfig speichert sich das Datenreihe unter einem Key in der Map. Und darum Überscheiben sich die beiden DataSeriesConfig.

Du siehst das auch wenn man das Dashboard wieder öffnet. Der Wert ist immer 100%.

Was tun? Man müsste den DataSeriesConfig noch mitgeben können unter welchem Key es sich die Datenreihe merken möchte. Zum Beispiel als ```DataSeries.Config.SECONDARY_DATA_SERIES```.

Reply:
Oh, habe mir den Fall des laden zwar angeschaut, aber wohl nicht genau genug. Die Serialisierung muss dementsprechend für eine Mehrfachauswahl an Datenreihen ebenfalls angepasst werden.

Reply:
Planst Du noch einen Commit hinterherzuschicken?

Reply:
Ja würde das schon noch gerne zu einem erfolgreichen Abschluss bringen. Habe aber gerade wenig Zeit für solche Projektchen. Wird noch etwas dauern, zumal ich mir jetzt nicht mehr ganz sicher bin wie ich das angehen soll. Bzw. hast du den Kommentar zum enum wieder entfernt?